test/electionguard/utils/byte_logger.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef __ELECTIONGUARD_CPP_TEST_UTILS_BYTE_LOGGER_HPP_INCLUDED__
+#define __ELECTIONGUARD_CPP_TEST_UTILS_BYTE_LOGGER_HPP_INCLUDED__
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+class ByteLogger
+{
+  public:
+    static void print(string msg, uint64_t *bignum, size_t size)
```

Comment:
these functions used to be in the core lib and are useful helpers but i put them in the utils instead since they are very much optional.